.circleci/config.yml

```diff
@@ -45,8 +45,6 @@ jobs:
             [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
             [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
             sudo mongod --config /etc/mongodb.conf &
-            paver test_system -t openedx/features/ucsd_features/tests/tests.py --fasttest
```

Comment:
Reason: These were written for custom contact_us view (to send email). Now we do not need it as we are using existing contact_us code.

Reply:
why we don't have any other tests under `ucsd_features`?

Reply:
Because we do not have made a lot of customizations other than caliper. And all caliper tests are now in the Caliper app. But we will add test cases for this send_email work.